src/include/distributed/multi_physical_planner.h

```diff
@@ -409,7 +405,8 @@ typedef struct DistributedPlan
 	 * contains the subplans generated part of the DistributedPlan.
 	 *
 	 * On the other hand, usedSubPlanNodeList keeps track of which subPlans
-	 * are used within this distributed plan.
+	 * are used within this distributed plan as a list of
+	 * UsedDistributedSubPlan pointers.
```

Comment:
As I'm getting used to having the same ids for different UsedsubPlan in the list change, I think it'd be nice to have the same explicit comment here:
```
* The list may contain duplicates if the subplan is referenced multiple
 * times.
```